base/bucket_gocb.go

```diff
@@ -987,6 +1135,68 @@ func (bucket CouchbaseBucketGoCB) WriteUpdate(k string, exp int, callback sgbuck
 
 }
 
+func (bucket CouchbaseBucketGoCB) WriteUpdateWithXattr(k string, xattr string, exp int, callback sgbucket.WriteUpdateWithXattrFunc) error {
```

Comment:
Forgot about that - thanks for the reminder.  I've updated here and in walrus and sg-bucket.